### DISCUSSION TITLE
레포지토리 이름 변경 제안

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#239] [General]
현재 레포지토리 이름 studygroup에서 다른 네이밍으로 변경하고 싶습니다!!
회사 다니면서 보니까 아예 관련 없는 단어를 선정하기도 하는데 (보안 이슈로 추정됩니다), 개인적으로 studygroup 뭔가 맘에 안듭니다 ㅋ

뜨끔 그대로 영어로 바꿔서 `ddeukkeum` 어떠세요? 좋은 이름 있으면 추천 한번 해주세요 ~

**Comment by Arkady1356** [comment]
ㅋㅋㅋㅋㅋㅋ 저는 너무 좋아요~
아무 이름을 추천해 보자면  

**cactusdy**:  
cactus + study + 사들람은 단에어서 순들서이 바어뀌도 이할해수있다

**Reply by rbgus2002** [reply]
아 웃개기네 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ짜진 장재우 같요아
캑터스디 좋다
